src/routes/contentReviews/index.js

```diff
@@ -102,120 +67,62 @@ const update = async (req, res) => {
   }
   const { id } = req.params;
   try {
-    const contentReview = await models.content_reviews.findOne({
+    const contentReview = await ReviewService.getReview(id);
+
+    const { text, is_spoiler, score } = req.body;
+    models.content_reviews.update({ text, is_spoiler, score }, {
```

Comment:
```suggestion
    await ReviewService.updateReview({ text, is_spoiler, score }, {
```
zaten halihazırda bir güncelleme servisimiz var, burada onu kullanmalıyız